flake/dev/all-maintainers.nix

```diff
@@ -0,0 +1,63 @@
+{ lib, ... }:
+{
+  perSystem =
+    { pkgs, config, ... }:
+    {
+      apps.all-maintainers = {
+        program = pkgs.writeShellApplication {
+          name = "update-all-maintainers";
+          runtimeInputs = with pkgs; [
+            git
+            coreutils
+          ];
+          text = ''
+            root="$(git rev-parse --show-toplevel)"
+            install -m 644 -T \
+              ${config.packages.all-maintainers} \
+              "$root/stylix/generated/all-maintainers.nix"
+          '';
+        };
+        meta.description = "update all-maintainers.nix";
+      };
+
+      packages.all-maintainers =
+        let
+          meta = pkgs.callPackage ../../stylix/meta.nix { };
+        in
+        pkgs.runCommand "all-maintainers"
+          {
+            passAsFile = [ "maintainers" ];
+            maintainers = lib.pipe meta [
+              builtins.attrValues
+              (builtins.concatMap (value: value.maintainers or [ ]))
+              (map (value: {
+                name = value.github;
+                inherit value;
+              }))
+              builtins.listToAttrs
+              (lib.generators.toPretty { })
+            ];
+            inherit (config.treefmt) projectRootFile;
+            nativeBuildInputs = [ config.formatter ];
+          }
+          ''
+            touch "$projectRootFile"
+
+            {
+              echo "# THIS FILE IS GENERATED with \`nix run .#all-maintainers\`, DO NOT EDIT"
+              # TODO: print the rest of the header
+              cat "$maintainersPath"
+            } >result.nix
+
+            treefmt --no-cache result.nix
+
+            install -m 644 -T result.nix "$out"
+          '';
+
+      checks.all-maintainers-up-to-date = pkgs.testers.testEqualContents {
```

Comment:
Hm, looks like CI is now unhappy with the file's owner/group?

```diff
--- /nix/store/xklkpwqshdyhzffp3bxsb9j2jci17h20-all-maintainers.nix
+++ /nix/store/m7f0q40i340xa3yaar7awrvr2gg813nc-all-maintainers
├── stat {}
│ @@ -1,7 +1,7 @@
│  
│    Size: 4085         Blocks: 8          IO Block: 4096   regular file
│ -Device: 1,20 Access: (0444/-r--r--r--)  Uid: (    0/    root)   Gid: (  350/  nixbld)
│ +Device: 1,20 Access: (0444/-r--r--r--)  Uid: (    0/    root)   Gid: (    0/   wheel)
```

https://github.com/nix-community/stylix/actions/runs/16252425101/job/45883909609?pr=1654#step:5:526